release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3325,6 +3325,63 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, enabling custom feature gates could cause the install to fail in AWS if the feature gate `ClusterAPIInstallAWS=true` was not enabled. With this release, `ClusterAPIInstallAWS=true` is no longer required. (link:https://issues.redhat.com/browse/OCPBUGS-36720[*OCPBUGS-36720*])
+
+* Previously, if `create cluster` was run after the `destroy cluster` command, an error would report that local infrastructure provisioning artifacts already exist. With this release, leftover artifacts are removed with `destroy cluster` and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36777[*OCPBUGS-36777*])
+
+* Previously, the OperandDetails displayed information for the first Custom Resource Definition (CRD) that matched by name. With this release, the OperandDetails page displays information for the CRD that matches by name and by the version of the operand. (link:https://issues.redhat.com/browse/OCPBUGS-36841[*OCPBUGS-36841*])
+
+* Previously, if the `openshift.io/internal-registry-pull-secret-ref` annotation was removed from a `ServiceAccount`, {product-title} recreated the deleted annotation and created a new managed image pull `Secret`. This contention could cause the cluster to get overloaded with image pull `Secrets`. With this release, {product-title} attempts to reclaim managed image pull `Secrets` that were previously referenced and deletes managed image pull `Secrets` that remain orphaned after reconciliation. (link:https://issues.redhat.com/browse/OCPBUGS-36862[*OCPBUGS-36862*])
+
+* Previously, some of the processes remained running after the Installer stopped due to setup failures. With this release, all installation processes stop when the Installer stops running. (link:https://issues.redhat.com/browse/OCPBUGS-36890[*OCPBUGS-36890*])
+
+* Previously, there was no runbook for `ClusterMonitoringOperatorDeprecatedConfig`. With this release, the runbook for `ClusterMonitoringOperatorDeprecatedConfig` alert is added and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36907[*OCPBUGS-36907*])
+
+* Previously, the *Cluster overview page* included a _View all steps in documentation_ link that resulted in a 404 error for ROSA and OSD clusters. With this update, the link does not appear for ROSA and OSD clusters. (link:https://issues.redhat.com/browse/OCPBUGS-37063[*OCPBUGS-37063*])
+
+* Previously, there was a mismatch between OpenSSL versions of MachineConfig Operator tools used by {product-title} and the OpenSSL version that runs on the Hosted Clusters Control Plane side. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37241[*OCPBUGS-37241*])
```

Comment:
Might want to double check this bug text with a hosted control planes writer. I know it's not capitalized, but am unsure about other details of how to discuss this offering.
```suggestion
* Previously, there was a mismatch between OpenSSL versions of Machine Config Operator tools used by {product-title} and the OpenSSL version that runs on the hosted control plane. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37241[*OCPBUGS-37241*])
```

Reply:
Good call - I'll check! TYSM :)

Reply:
As per https://docs.openshift.com/container-platform/4.16/architecture/control-plane.html#hosted-control-planes-benefits_control-plane, looks like you're right! TY!